src/PrestaShopBundle/Tests/Translation/Extractor/ThemeExtractorTest.php

```diff
@@ -0,0 +1,96 @@
+<?php
+/**
+ * 2007-2016 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ *  @author 	PrestaShop SA <contact@prestashop.com>
+ *  @copyright  2007-2016 PrestaShop SA
+ *  @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ *  International Registered Trademark & Property of PrestaShop SA
+ */
+
+namespace PrestaShopBundle\Tests\Translation\Extractor;
+
+use Symfony\Component\Yaml\Yaml;
+use Symfony\Component\Filesystem\Filesystem;
+use Symfony\Bundle\FrameworkBundle\Test\KernelTestCase;
+
+use PrestaShop\PrestaShop\Core\Addon\Theme\Theme;
+
+class ThemeExtractorTest extends KernelTestCase
+{
+    const THEME_ROOT_DIR = __DIR__.'/../../resources/themes/fake-theme';
```

Comment:
because ~30% of PrestaShop installations are still done using PHP 5.4.

Help us, use fresh servers with PHP7 for your customers. Can't wait to remove this constraint :/